finetuner/data.py

```diff
@@ -211,6 +242,13 @@ def create_document(
                 doc.load_uri_to_blob()
         else:
             raise ValueError(f'Expected uri in column 1, got {column}')
+    elif modality == 'mesh':
+        if _is_uri(column):
+            doc = Document(uri=column)
+            if create_point_clouds and os.path.isfile(column):
+                doc.load_uri_to_point_cloud_tensor(point_cloud_size)
```

Comment:
this document would be unusable if `create_point_clouds` is False but `column` is a local file right?

Reply:
yes more or less, but it is the same for images when you set `convert_to_blob=False`.

Reply:
good point, maybe both of these cases should raise an error or warning